django/db/backends/oracle/utils.py

```diff
@@ -23,7 +24,7 @@ class InsertVar:
         "FloatField": Database.NATIVE_FLOAT,
         "DateTimeField": Database.TIMESTAMP,
         "DateField": Database.Date,
```

Comment:
Do you know if any of these remaining three are also necessary? Does it work on `oracledb` but not `cx_Oracle`?

Reply:
> Do you know if any of these remaining three are also necessary?

Yes, they are. You can find corresponding tests in `queries.test_db_returning.ReturningValuesTests`.

> Does it work on `oracledb` but not `cx_Oracle`?

It doesn't work on both :smile:

Reply:
... and I there is ticket-30170 assigned to myself :see_no_evil: